packages/v2ray/common-platform-others.go.patch

```diff
@@ -1,15 +0,0 @@
---- a/common/platform/others.go
-+++ b/common/platform/others.go
-@@ -31,9 +31,9 @@
- 	defPath := filepath.Join(assetPath, file)
```

Comment:
It uses `adrg/xdg` package to search for pathes, in which pathes are hard-coded. I think extra patches on `adrg/xdg` are needed.